packages/styleguide/src/components/Button/Button.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { oneOf } from 'prop-types';
+import styled from 'styled-components';
+
+import { rem } from './../../style/utils';
+
+const propTypes = {
+  variant: oneOf(['code-toggle'])
```

Comment:
Toto stale nie je podla mna vseobencne, je to menom sponjene s code togglom. Preto by to nemal byt vseobecny variant, lebo sa neda vseobencne pouzit.